src/Behat/Behat/Transformation/Transformer/RepositoryArgumentTransformer.php

```diff
@@ -180,6 +200,35 @@ private function isApplicableTableTransformation(Transformation $transformation,
     }
 
     /**
+     * Checks if provided transformation is applicable row table transformation.
+     *
+     * @param Transformation $transformation
+     * @param mixed          $value
+     *
+     * @return Boolean
+     */
+    private function isApplicableRowTableTransformation(Transformation $transformation, $value)
+    {
+        if (!$value instanceof TableNode) {
+            return false;
+        };
+
+        try {
```

Comment:
This logic is a bit weird. Should I move this back to the NodeTable in Gherkin, and add a "Column count" method or is it okay as it is?